tweets/next-10-summit2.tweet

```diff
@@ -0,0 +1,5 @@
+The Next-10 team is planning a mini-summit November 18 to do a deep
+dive on Types for Node.js developers and on building single executable
+binaries with Node.js. Full details in https://github.com/nodejs/next-10/issues/99.
+If you have thoughts/opinions on these topics and want to help
+Node.js continue to be successful we hope to see you there.
```

Comment:
For tweets, please don't do hard line breaks. They'll transfer to the tweet which is not a desirable effect for a number of reasons.